docs/features.rst

```diff
@@ -761,3 +761,12 @@ The gist command creates a gist comprised of:
 
 The URL for the gist is displayed on the terminal of the user as output, and a web browser will automatically open a tab to the gist.
 
+Seeing Stack Traces Automatically
+---------------------------------
+If you would like to investigate bugs in CumulusCI when you find them, you can set the config 
+option `show_exceptions` to `True` in the `cli` section of `~/.cumulusci/cumulusci.yml` and exceptions will no 
+longer be surpressed when they are thrown within CumulusCI. Usage Errors (wrong command 
```

Comment:
I think the option should be called something like `show_exception_stack_traces`; we are never suppressing the exception message.

Also: spelling of suppressed

Reply:
d3b626236dff6c3f40a2e13b9f5115d46607d21a